wordpress-mobile/WordPress-iOS
# Apply Default Values to the CoreData items added in 117

Fixes a migration issue that was a result of changes made when adding some values to the 117 Data Model.

## To test:
1. Install a previous build such as one from `trunk`
2. Navigate to the page creation screen where you can pick starter content
    - This will populate entities that will be migrated in the next step
3. Install a build from this branch
4. **Expect** the Migration to be successful
5. Navigate to the page creation screen where you can pick starter content
6. **Expect** to not have any issues loading that page

## PR submission checklist:
